dist/services.html

```diff
@@ -0,0 +1,140 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea! - Services</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+
+  <div class="container">
+    <!-- Page header -->
+    <header class="header">
+      <nav class="navigation">
+        <a href="./services.html">Services</a>
+        <a href="#product">Product</a>
+        <a href="#vision">Vision</a>
+        <a href="#features">Features</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+      </nav>
+      <a href="./index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+    </header>
+
+    <!-- Introduction -->
+    <section class="svc-section services-intro">
+      <div class="img-wrapper">
+        <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+      </div>
+      <div class="services-into-text">
+        <h1>Services</h1>
+        <p>
+          Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum,
+          accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+          consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+      </div>
+    </section>
+
+   <!-- List of services  -->
+    <section class="svc-section services-list">
+      <div class="service-item">
+        <h2 class="service-name">Digital Design</h2>
+        <p class="service-description">
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+          consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <div class="button svc-button">Lean More</div>
```

Comment:
Wilfred, your code looks incredible yet again. I'm looking for something to nitpick, and I finally found it. There's a typo! One of your buttons says "Lean More," instead of "Learn More." 

So I know that the stretch problems are optional, but I know you can handle them. The stretch problems are there for people like you, who are more advanced. I looked at your website, and your Github repos. You've already built API's and React apps! I hope you'll start reaching for the stretch challenges on each project, to push your knowledge even further. They often introduce us to excellent technology in the stretch problems that they don't have time to cover in the normal curriculum.

Reply:
Thanks for the feedback Joe. The typo has been fixed. I would've attempted the stretch goals but I ran out of time before I had to submit the pull report for our stand up meeting.

Yes I've built APIs and React apps but CSS is still the most difficult and time consuming part of the development process. The instruction I've received about box model, relative/absolute/fixed positioning, and flexbox has really made the CSS dev process easier.  This project took me 2 hours to complete from start to finish whereas it would've taken me 8 - 10 hours.

I'll work on some of the stretch goals and ping you so you can check it out.

Thanks again.